packages/datagrid/src/containers/datagrid.container.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Component from '../components/';
+import DATAGRID_PROPTYPES from '../components/datagrid.proptypes';
+
+export default function DataGrid(props) {
+	const onFocusedCell = event => {
+		if (props.onFocusedCell) {
+			props.onFocusedCell(event);
+		}
+
+		if (props.onFocusedCellActionCreator) {
```

Comment:
please check https://github.com/Talend/ui/pull/1097 ;)

Reply:
This pr is not merged, i'm waiting.